src/typedKnex.ts

```diff
@@ -97,12 +99,12 @@ export interface ITypedQueryBuilder<Model, SelectableModel, Row> {
 
     whereParentheses: IWhereParentheses<Model, SelectableModel, Row>;
 
-    groupBy: ISelectableColumnKeyFunctionAsParametersReturnQueryBuider<Model, SelectableModel, Row>;
+    groupBy: IColumnParameterNoRowTransformation<Model, SelectableModel, Row>;
 
     having: IHaving<Model, SelectableModel, Row>;
 
-    havingNull: ISelectableColumnKeyFunctionAsParametersReturnQueryBuider<Model, SelectableModel, Row>;
-    havingNotNull: ISelectableColumnKeyFunctionAsParametersReturnQueryBuider<Model, SelectableModel, Row>;
+    havingNull: IColumnParameterNoRowTransformation<Model, SelectableModel, Row>;
```

Comment:
This one is identical to `ISelectableColumnKeyFunctionAsParametersReturnQueryBuider` so I removed it. Let me know if I should revert this, but you don't seem to use it anywhere else